src/iconsManifest/supportedExtensions.ts

```diff
@@ -4391,11 +4391,8 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
     },
     {
-      icon: 'truffle',
```

Comment:
It seems you removed the `truffle` icon?

Reply:
Sorry for that! Fixed.